cli/flags.rs

```diff
@@ -199,6 +209,7 @@ pub fn parse_flags(matches: ArgMatches) -> DenoFlags {
 
     flags.v8_flags = Some(v8_flags);
   }
+  flags.config_path = matches.value_of("var").map(ToOwned::to_owned);
```

Comment:
Is this intended?

Reply:
Alas, my bad... Nope... (I guess I was kinda sleepy now...)